docs/qmlstart/qmlstart.rst

```diff
@@ -130,7 +130,7 @@ Basic Elements
 
 Elements can be grouped into visual and non-visual elements. A visual element (like the ``Rectangle``) has a geometry and normally presents an area on the screen. A non-visual element (like a ``Timer``) provides general functionality, normally used to manipulate the visual elements.
 
-Currently, we will focus on the fundamental visual elements, such as ``Item``, ``Rectangle``, ``Text``, ``Image`` and ``MouseArea``.
+Currently, we will focus on the fundamental visual elements, such as ``Item``, ``Rectangle``, ``Text``, ``Image`` and ``MouseArea``. However, by using the Qt Quick Controls 2.0 module, it is possible to create user interfaces built from standard platform components such as buttons, labels and sliders. By using the Quick Controls module, your user interface will automatically adapt to the platform it is running on and look like any other native application.
```

Comment:
Which module are you referring to with "Qt Quick Controls module"? For Controls 1, the statement "your user interface will automatically adapt to the platform it is running on and look like any other native application" makes sense for desktop platforms, but not Qt Quick Controls 2.

I don't know if we really want to mention Controls 1, but if we do, it should be specific by mentioning the version number to avoid confusion. If you're referring to Controls 2, I would probably just remove the last sentence here.

Reply:
Thx! I will update accordingly.